server/src/lib/model/user-client.ts

```diff
@@ -65,6 +65,39 @@ async function updateLocales(
   }
 }
 
+async function updateDemographics(
+  clientId: string,
+  age?: string,
+  sex?: string
+) {
```

Comment:
Where did we land on making sure that age/sex aren't null before we process the rest of this?

Reply:
Done! Added in https://github.com/mozilla/common-voice/pull/2868/commits/499fc75803678b41374aa6b0cabd2d620dd85a8e :)